tests/pytests/scenarios/compat/test_with_versions.py

```diff
@@ -43,7 +43,7 @@ def _get_test_versions_ids(value):
 
 
 @pytest.fixture(
-    params=("3002.0", "3003.0", "3004.0"), ids=_get_test_versions_ids, scope="module"
+    params=("3002.8", "3003.3", "3004"), ids=_get_test_versions_ids, scope="module"
```

Comment:
Should these be 3002, 3003 and 3004 ?
since pinning them to 3002.8, 3003.3 ->.4 ? and 3004 -> 3004.1 ? these could change with a CVE or other issue

Reply:
As far as I know, this is a temporary fix.  I'm proposing this now just to get others unblocked.

Reply:
so my comment is wondering about the versions you have picked

3003.3 and 3004 are CVE affected releases and now reside on the archive repo.

Reply:
suggest 3002, 3003 and 3004 which would pick latest major versions

Reply:
The test is supposed to test the latest minor version of each major, it's only being changed because the 3004.1 changes haven't landed yet.
Why would we want to only test major versions?

Reply:
didn't know the test was for minor versions only